data/tokens.json

```diff
@@ -3021,6 +3021,253 @@
       "type": "spacing"
     }
   },
+  "inputs": {
+    "gtmds_input_size-height-sm": {
+      "value": "{gtmds_size-lg}*2",
+      "type": "sizing"
+    },
+    "gtmds_input_size-height-lg": {
+      "value": "{gtmds_size-xl}+{gtmds_size-lg}",
+      "type": "sizing"
+    },
+    "gtmds_input_size-width": {
+      "value": "auto",
+      "type": "sizing"
+    },
+    "gtmds_input_margin-right": {
+      "value": "{gtmds_space-xxs}",
+      "type": "spacing"
+    },
+    "gtmds_input-content_margin-right": {
+      "value": "{gtmds_space-xxs}/2",
+      "type": "spacing"
+    },
+    "gtmds_input_margin-bottom": {
+      "value": "{gtmds_space-xxs}/2",
+      "type": "spacing"
+    },
+    "gtmds_input_padding-left": {
+      "value": "{gtmds_space-xxs}",
+      "type": "spacing"
+    },
+    "gtmds_input_padding-right": {
+      "value": "{gtmds_space-xxs}",
+      "type": "spacing"
+    },
+    "gtmds_input_padding-vertical-sm": {
+      "value": "{gtmds_space-xxs}/2",
+      "type": "spacing"
+    },
+    "gtmds_input_padding-vertical-lg": {
+      "value": "{gtmds_space-xxs}",
+      "type": "spacing"
+    },
+    "Chip": {
+      "gtmds_input_margin-row-bottom": {
+        "value": "{gtmds_space-xxs}/2",
+        "type": "spacing"
+      },
+      "gtmds_input_margin-row-right": {
+        "value": "{gtmds_space-xxs}/2",
+        "type": "spacing"
+      },
+      "gtmds_input_padding-left-sm": {
+        "value": "{gtmds_space-xxs}/2",
+        "type": "spacing"
+      }
+    },
+    "Invisible-box": {
+      "gtmds_input_padding-right": {
+        "value": "0",
+        "type": "spacing"
+      }
+    },
+    "Background": {
```

Comment:
I see you split colors into categories, we only do that for variants of a component or to set apart individual components that have been aggregated see tokens from the selection controls set. Let's discuss if we should have it all flattened tomorrow morning.